README.md

```diff
@@ -1,8 +1,7 @@
 ## Introduction
 
-<!-- TODO nf-core: Write a 1-2 sentence summary of what data the pipeline is for and what it does -->
-
-**Arcadia-Science/seqqc** is a bioinformatics best-practice analysis pipeline for Quality control for sequencing data.
+**Arcadia-Science/seqqc** is a bioinformatics best-practice analysis pipeline for quality control for sequencing data.
+The pipeline can be used on short or long read sequencing data (in FASTQ format) to identify common problems like the presence of adapters, high sequencing duplication rates, mislabelled samples, and contamination.
```

Comment:
Some questions for long-read implementations: 
1. From what I remember, the immediate deliverable for PacBio isn't fastq sometimes. Are we adding upstream steps for converting between file formats to get to fastq or depending on somebody else to do that prior to the QC workflow? 
2. The last time I did Nanopore sequencing, you have a bunch of small fastq/fast5 files from basecalling and when you do the adapter removal with a tool like porechop then it's a single fastq. Are we assuming that those steps are being done prior to Nanopore reads going through this workflow?

Reply:
I'm going to open an issue with these questions. I'm envisioning that this pipeline would work on any fastq data, but I think that can be a conversation and it can always evolve if we need it to.